pkg/sign/signer.go

```diff
@@ -149,7 +148,7 @@ func Bundle(content Content, keypair Keypair, opts BundleOptions) (*protobundle.
 			return nil, err
 		}
 
-		artifactOpts := verify.WithArtifact(bytes.NewReader(content.PreAuthEncoding()))
+		artifactOpts := verify.WithoutArtifactUnsafe()
```

Comment:
Can we add a comment that says we are just doing a signature check, and that this isn’t safe to do for verification, or something similar?